src/pages/Marketplace/Cards/Card.tsx

```diff
@@ -16,7 +16,7 @@ export default function Card({
   name,
   image,
   id,
-  endow_type,
+  endow_designation,
```

Comment:
Just noticed we have both [EndowDesignation](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/types/aws/ap/index.ts#L90) and [EndowmentType](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/types/contracts/common.ts#L51), what will be the difference between the two?

Reply:
> Just noticed we have both [EndowDesignation](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/types/aws/ap/index.ts#L90) and [EndowmentType](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/types/contracts/common.ts#L51), what will be the difference between the two?

`EndowmentType` is the Contract level endowment differentiation (`Charity` vs `Normal`), whereas `EndowDesignation` is a purely AWS DB field(string type) that further differentiates the Charity endowments.